resources/seccomp/aarch64-unknown-linux-musl.json

```diff
@@ -170,6 +170,19 @@
                     }
                 ]
             },
+            {
+                "syscall": "futex",
+                "comment": "Used for synchronization",
+                "args": [
+                    {
+                        "index": 1,
+                        "type": "dword",
+                        "op": "eq",
+                        "val": 137,
+                        "comment": "FUTEX_WAIT_BITSET_PRIVATE"
+                    }
+                ]
+            },
```

Comment:
it is still used. which I actually found out manually. I want to work on a test that

1. makes sure that we do not have holes in our seccomp filters (so that we can catch issues faster), and
2. makes sure that our filters are not too permissive (that's a bit trickier I think).